artistools/inputmodel/from_alcar.py

```diff
@@ -132,14 +184,16 @@ def get_grid() -> tuple[
     nu = 2
     # ... cylindrical coordinates of the particle positions
     rcyltraj, zcyltraj = np.zeros(ntraj), np.zeros(ntraj)
-    for i in np.arange(ntraj):
+    for i in [int(j) for j in np.arange(ntraj)]:
```

Comment:
I don't understand what is going on here that would make this necessary. range(ntraj) should give you an iterable of ints.